public/scripts.js

```diff
@@ -40,6 +44,15 @@ document.addEventListener('DOMContentLoaded', function() {
     var blockquote = querySelector('blockquote');
     var content = querySelector('.quote');
 
+    if (showTalkingHeads) {
+      hideTalkingHead();
+      setTimeout(function() {
+        showTalkingHead(quote.author, quote.quote);
+      }, 1000);
+    } else {
+      hideTalkingHead();
+    }
```

Comment:
Perhaps we could simplify this entire `if/else` by moving the hiding outside and removing the `else` part?

```javascript
hideTalkingHead();

if (showTalkingHeads) {
  setTimeout(() => ...);
}
```